tests/apps/courses/test_models_course.py

```diff
@@ -529,120 +530,85 @@ def test_models_course_get_persons_other_placeholders(self):
 
     def test_models_course_get_course_runs_empty(self):
         """
-        For a course without course runs the methods `get_course_runs` and
-        `get_course_runs_for_language` should return an empty query.
+        For a course without course runs the methods `get_course_runs`should
```

Comment:
```suggestion
        For a course without course runs the methods `get_course_runs` should
```

Reply:
Lol it looks like I have a space bar problem with my keyboard ^^